src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/HttpTransportSecurityElement.cs

```diff
@@ -44,5 +44,18 @@ public string Realm
                 base[ConfigurationStrings.Realm] = value;
             }
         }
+
+        internal void ApplyConfiguration(HttpTransportSecurity security)
+        {
+            if (security == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(nameof(security));
+            }
+
+            security.ClientCredentialType = ClientCredentialType;
+           // security.ProxyCredentialType = this.ProxyCredentialType;
+            security.Realm = Realm;
```

Comment:
ProxyCredentialType is a client side property only so it's not a case of we haven't implemented this yet, we will never add that property. It's safe to remove this. Leave ExtendedProtectionPolixy though, we will need to add that eventually